### PR TITLE
docs: improve the prerequisites of RWX volume

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
improve the prerequisites of the RWX volume section

### CODE REVIEW

File: docs/rancher/csi-driver.md [hunk 1]
```diff
@@ -381,6 +381,25 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
 
   ![](/img/v1.4/rancher/harvester-csi-driver-version.png)
 
+- Each nodes in the guest cluster needs to have the NFS client installed.
```
Comment (bk201) [inline docs/rancher/csi-driver.md:384]:
```suggestion
- Each node in the guest cluster must have the NFS client installed.
```

File: docs/rancher/csi-driver.md [hunk 2]
```diff
@@ -381,6 +381,25 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
...
+
+  - For CentOS and RHEL:
+  ```
+  yum install -y nfs-utils
+  ```
+
+  - For SUSE/OpenSUSE:
+  ```
+  zypper install -y nfs-client
+  ```
```
Comment (jillian-maroket) [inline docs/rancher/csi-driver.md:401]:
```suggestion
- The NFS client is installed on each node in the guest cluster.

  Run any of the following commands to install the NFS client.

  - Debian and Ubuntu: `apt-get install -y nfs-common`
  
  - CentOS and RHEL: `yum install -y nfs-utils`

  - SUSE and OpenSUSE: `zypper install -y nfs-client`
```

**Review by jillian-maroket — COMMENTED** [review]
Just a few nits

File: docs/rancher/csi-driver.md (hunk 2 above)
Comment (Vicente-Cheng) [reply docs/rancher/csi-driver.md]: Thanks for the suggestion!  Updated!

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by bk201 — APPROVED** [review]

**Pull request merged**